plugin/web/components/common/Icon/icons.ts

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/order */
+
+// Misc
+import Logo from "./Icons/reearthLogo.svg";
```

Comment:
Since we aren't using the Icon component any more, this component should be modified.
It should be:
- renamed to index.ts
- export all icons like:
```
export {
Trash,
PlateauLogo,
...
...
}
```

Reply:
done

Reply:
https://github.com/eukarya-inc/reearth-plateauview/pull/29/commits/326aa3062b26c0e076d4cc56cf8ad8237c0cf1e2